include/glow/Base/Type.h

```diff
@@ -113,6 +113,37 @@ struct ShapeNCHW {
   }
 };
 
+struct ShapeTLBR {
+  /// Top padding.
+  size_t top;
+  /// Left padding.
+  size_t left;
+  /// Bottom padding.
+  size_t bottom;
+  /// Right padding.
+  size_t right;
+
+  explicit ShapeTLBR(llvm::ArrayRef<size_t> shape) {
+    assert(shape.size() == 4 && "Invalid padding");
+    top = shape[0];
+    left = shape[1];
+    bottom = shape[2];
+    right = shape[3];
+  }
+
+  explicit ShapeTLBR(size_t top, size_t left, size_t bottom, size_t right)
```

Comment:
I do not see this ctor is used.

Reply:
You are right. It was used in creating the zero TLBR padding, which is deleted according your suggestion.